webview-ui/src/components/settings/ModelPicker.tsx

```diff
@@ -103,7 +103,7 @@ export const ModelPicker = ({
 
 	const { id: selectedModelId, info: selectedModelInfo } = useSelectedModel(apiConfiguration)
 
-	const [searchValue, setSearchValue] = useState(selectedModelId || "")
+	const [searchValue, setSearchValue] = useState("") // kilocode_change
```

Comment:
Please add a comment that you removed something, otherwise it will look in the merge like roo added something